File: C-C++/MatrixMultiplacation.c
#include <stdio.h>

int main(void) {
   int p,q,r,s;
   //get the row and columns of the matrix
   printf("Enter the rows and columns of the first matrix : ");
   scanf("%d %d",&p,&q);
   printf("Enter the rows and columns of the second matrix : ");
   scanf("%d %d",&r,&s);
   //check for the eligblity of the matrix for multiplacation
   if (q != r){
        printf("The matrix is not eligible for multiplacation");
        return 0;
   }
   //get matrix values
   int m[p][q], n[r][s];
   //first matrix
   printf("Enter the value of the first matrix : \n");
   for (int i=0; i< p; i++) {
        for (int j=0; j< q; j++) {
            printf("Enter the value for %d %d : ",i,j);
            scanf("%d",&m[i][j]);
        }
   }
   //second matrix
   printf("Enter the value of the second matrix : \n");
   for (int i=0; i< r; i++) {
        for (int j=0; j< s; j++) {
            printf("Enter the value for %d %d : ",i,j);
            scanf("%d",&n[i][j]);
        }
   }
   //multiplay the matrix
   int res[p][s],result;
   for (int i = 0; i < p; i++){
        for (int j = 0; j < s; j++){
            result = 0;
            for (int k = 0; k < q; k++){
                result += m[i][k] * n[k][j];
            }
            res[i][j] = result;
        }
   }
   //print the result
    printf("The result is :\n");
    for (int i=0; i< p; i++) {
        for (int j=0; j< s; j++) {
            printf("%5d",res[i][j]);
        }
        printf("\n");
   }
}

File: Python/matrixMultiplacation.py
import sys
#geting the number of rows and columns
p = int(input("Enter the number of rows of the first matrix : "))
q = int(input("Enter the number of columns of the first matrix : "))
r = int(input("Enter the number of rows of the second matrix : "))
s = int(input("Enter the number of columns of the second matrix : "))
#checking the eligiblity of the matrix for multiplication
if q != r:
    print("The matrix is not eligible for multiplication")
    sys.exit(0)
#getting the values of the matrix
m = [[int(input("Enter the value for first matrix : ")) for i in range(q)] for j in range(p)]
n = [[int(input("Enter the value for second matrix : ")) for i in range(s)] for j in range(r)]
#show the result
for i in range(p):
    for j in range(s):
        result = 0
        for k in range(q):
            result += m[i][k] * n [k][j]
        print(result, end="  ")
    print()